ckanext/ckanext-apicatalog_routes/ckanext/apicatalog_routes/plugin.py

```diff
@@ -103,6 +103,15 @@ def get_dataset_labels(self, dataset_obj):
             except ObjectNotFound:
                 continue
 
+        pkg_dict = get_action('package_show')(context, {'id': dataset_obj.id})
+
+        if pkg_dict.get('access_restriction_level') and \
+                pkg_dict.get('access_restriction_level') == 'only_allowed_organizations':
+            allowed_organizations = pkg_dict.get('allowed_organizations', "").split(',')
```

Comment:
Fails with spaces after commas. Suggestion:
```suggestion
            allowed_organizations = [o.strip() for o in pkg_dict.get('allowed_organizations', "").split(',')]
```